lib/stat_tracker.rb

```diff
@@ -0,0 +1,84 @@
+require "csv"
+
+class StatTracker
+
+  def self.from_csv(locations)
+    StatTracker.new(locations)
+  end
+
+  attr_reader :games, :teams, :game_teams
+
+  def initialize(locations)
+    @games = create_games(locations[:games])
+    @teams = create_teams(locations[:teams])
+    @game_teams = create_game_teams(locations[:game_teams])
+  end
+
+# helper methods
+  def csv_reader(file_path)
+    CSV.parse(File.read(file_path), headers: true, header_converters: :symbol)
+  end
+
+  def create_games(file_path) 
+    data = csv_reader(file_path)
+    games = []
+    data.each do |row|
+      games << Game.new(
+        row[:game_id],
+        row[:season], 
+        row[:type], 
+        row[:date_time], 
+        row[:away_team_id], 
+        row[:home_team_id],
+        row[:away_goals],
+        row[:home_goals],
+        row[:venue],
+        row[:venue_link]
+      )
+    end
+    games
+  end
+
+  def create_teams(file_path) 
+    data = csv_reader(file_path)
```

Comment:
rename `csv_reader` to `read_csv` to keep method names as verbs, not nouns

```suggestion
    data = read_csv(file_path)
```